src/Functions/SFA.DAS.Apprenticeships.Approvals.EventHandlers.Functions.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.1.3" />
     <PackageReference Include="NServiceBus" Version="7.7.4" />
-    <PackageReference Include="SFA.DAS.CommitmentsV2.Messages" Version="8.29.1" />
+    <PackageReference Include="SFA.DAS.CommitmentsV2.Messages" Version="100.0.1" />
```

Comment:
approving on the assumption this is the agreed way forward with this package versioning issue